src/scripts/marker.js

```diff
@@ -99,10 +99,10 @@ export class Marker {
                 hidden: token.data.hidden,
                 locked: false,
                 flags: {deckMarker: true}
-            });
+            }]);
 
             if (game.user.isGM) {
-                await canvas.scene.createEmbeddedEntity('Tile', newTile.data);
+                await canvas.scene.createEmbeddedDocuments('Tile', newTile.data);
```

Comment:
Not actually sure this is doing anything here, since I don't think we get newTile back. and I don't think newTile.data is right for grabbing the data?